bigbone/src/test/kotlin/social/bigbone/api/method/NotificationMethodsTest.kt

```diff
@@ -33,13 +33,13 @@ class NotificationMethodsTest {
     }
 
     @Test
-    fun getReportNotification() {
+    fun getReblogNotification() {
         val client = MockClient.mock("notifications.json")
         val notificationMethods = NotificationMethods(client)
         val pageable = notificationMethods.getAllNotifications().execute()
 
         val reportNotification = pageable.part[1]
-        reportNotification.type shouldBeEqualTo "admin.report"
+        reportNotification.type.name.lowercase() shouldBeEqualTo "reblog"
         reportNotification.report shouldNotBe null
         reportNotification.report?.id shouldBeEqualTo "48914"
         reportNotification.report?.actionTaken shouldBeEqualTo false
```

Comment:
We should probably rename `reportNotification` to `reblogNotification` after this change from `admin.report` to `reblog`.